tiledb/sm/query/readers/sparse_unordered_with_dups_reader.cc

```diff
@@ -1395,58 +1395,29 @@ Status SparseUnorderedWithDupsReader<BitmapType>::copy_fixed_data_tiles(
 }
 
 template <class BitmapType>
-std::vector<uint64_t>
-SparseUnorderedWithDupsReader<BitmapType>::compute_fixed_results_to_copy(
-    const std::vector<std::string>& names,
+tuple<bool, std::vector<uint64_t>>
+SparseUnorderedWithDupsReader<BitmapType>::compute_fixed_result_tiles_to_copy(
```

Comment:
IMO this function name would be clearer as `resize_fixed_result_tiles_to_copy` because it mutates the argument.